Enable Raspberry Pi CSI2 cameras (#50)
groundlight/framegrab: README.md

```

## Optional Dependencies
Certain camera types have additional dependencies that must be installed separately. If you don't intend to use these camera types, you don't need to install these extra packages. 

- To use a Basler USB or GigE camera, you must separately install the `pypylon` package.
- To use Intel RealSense cameras, you must install `pyrealsense2`.

> - To use a CSI2 camera connected with a ribbon cable to a Raspberry Pi, you must install `python3-picamera2`. See install instructions at the [picamera2 github repository](https://github.com/raspberrypi/picamera2).

Reviewer: ```suggestion
- To use a Raspberry PI "CSI2" camera (connected with a ribbon cable), you must install the `picamera2` library.  See [pip](https://pypi.org/project/picamera2/) or the [picamera2 github repository](https://github.com/raspberrypi/picamera2).
```

Author: I specified that github repository because pip is not the recommended way to install `picamera2`. In order for it to work for me, I had to install it with apt (`sudo apt install -y python3-picamera2 --no-install-recommends`) and update the venv creation line in `groundlight-pi-gen` to include `--system-site-packages`

```
### Configurations
The table below shows all available configurations and the cameras to which they apply.
| Configuration Name         | Example         | Generic USB     | RTSP      | Basler    | Realsense | Raspberry Pi CSI2 |
|----------------------------|-----------------|------------|-----------|-----------|-----------|-----------|
| name                       | On Robot Arm    | optional   | optional  | optional  | optional  | optional  |
| input_type                 | generic_usb    | required   | required  | required  | required  | required  |

> | id.serial_number           | 23458234       | optional   | -         | optional  | optional  | optional  |

Reviewer 2: Do Raspberry Pi CSI2 cameras actually have a serial number that we can recognize? When you create a grabber and then print its config, do you see a serial number? Other grabber types such as Generic USB will discover the serial number (linux only). Also, you can provide a serial number to disambiguate between different cameras. Since this new grabber type only supports one grabber, providing a serial number wouldn't be necessary (except maybe to differentiate between a USB cam that might be confused for a Raspberry Pi CSI2. In any case, if we cannot discover or use the serial numbers for these cameras, maybe this chart should say `-` for id.serial_number for Raspberry Pi CSI2.
